### PR TITLE
Use KernelAbstractions 0.5

### PR DESCRIPTION
**Pull request by DilumAluthge**
@mcabbott Try this out and see if it fixes the KernelAbstraction tests in Tullio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div align="center">
-<h1>Tullio.jl</h1>
+<h1>Tullio.jl</h1>
```
Comment (mcabbott) [inline README.md:2]:
```suggestion
<h1>Tullio.jl</h1>
```

File: Project.toml [hunk 2]
```diff
@@ -13,7 +13,7 @@ CUDA = "1, 2"
 DiffRules = "1"
 FillArrays = "0.10"
 ForwardDiff = "0.10"
-KernelAbstractions = "0.4"
+KernelAbstractions = "0.5"
```
Comment (mcabbott) [inline Project.toml:16]:
```suggestion
KernelAbstractions = "0.4, 0.5"
```

**Comment by mcabbott** [comment]
Weirdly the one failure on last run was "NIGHTLY/t-2/group-3", whereas KA is tested in group 2.

**Comment by DilumAluthge** [comment]
@mcabbott Looks like this PR fixes group 2. Is this good to merge?

**Comment by DilumAluthge** [comment]
Bump @mcabbott

**Pull request closed without merging**